docs/documentation_routes.js

```diff
@@ -17,6 +17,11 @@ router.get('/install', function (req, res) {
   res.render('install', { 'releaseURL': url })
 })
 
+router.get('/download-kit-redirect', function (req, res) {
```

Comment:
This function already exists in server.js (line 150). Happy moving here if you think it's a better place, but should remove the other and make sure links work.

What do you think of calling it `download-kit-latest`?